docs/docsite/rst/network/user_guide/validate.rst

```diff
@@ -0,0 +1,164 @@
+.. _validate_data:
+
+*************************************************
+Validate data against set criteria with Ansible
+*************************************************
+
+The :ref:`validate <ansible_collections.ansible.utils.validate_module>` module validates data against your predefined criteria using a validation engine. You can pull this data from a device or file, validate it against your defined criteria, and use the results to identify configuration or operational state drift and optionally take remedial action.
+
+
+.. contents::
+   :local:
+
+Understanding the validate plugin
+==================================
+
+The `ansible.utils <https://galaxy.ansible.com/ansible/utils>`_ collection includes the :ref:`validate <ansible_collections.ansible.utils.validate_module>` module.
+
+To validate data:
+
+#. Pull in structured data or convert your data to structured format with the :ref:`cli_parse <ansible_collections.ansible.utils.cli_parse_module>` module.
+#. Define the criteria to test that data against.
+#. Select a validation engine and test the data to see if it is valid based on the selected criteria and validation engine.
+
+The structure of the data and the criteria depends on the validation engine you select. The examples here use the ``jsonschema`` validation engine provided in the `ansible.utils <https://galaxy.ansible.com/ansible/utils>`_ collection.
```

Comment:
Can add a mention that Red Hat Ansible Automation Platform subscription supports limited use if `jsonschema` public APIs as documented.